web/src/components/FacetToggle.tsx

```diff
@@ -37,7 +37,7 @@ const FacetToggle: React.FC<FacetToggleProps> = ({
 
   return (
     <FilterFacet>
-      <Tooltip text={tooltip} flow='up'>
+      <Tooltip text={tooltip} flow='up' trigger='hover'>
```

Comment:
Could we just use `trigger='hover' as default to avoid those changes? Apparently its also definitely the default and most common use case for our tooltips.